packages/docs/pages/Alert/AlertPage.tsx

```diff
@@ -12,121 +12,149 @@ import { AlertPropTable } from '../../PropTables/AlertPropTable';
-      <Code>warning</Code>, <Code>success</Code>, <Code>info</Code>.
-    </Text>
-
-    <Text>
-      To use this component, put it in your root component (e.g. place it after <Code>GlobalStyles</Code> component):
-    </Text>
-
-    <CodeSnippet>
-      {`
-        export const alertsManager = createAlertsManager(); // import this in child components to use alerts
-        
-        function App() {
-          return (
-            <>
-              {/* ... */}
-              <GlobalStyles />
-              <AlertsManager manager={alertsManager} />
-              {/* ... */}
-            </>
-          );
-        }
-      `}
-    </CodeSnippet>
-
-    <Text>
-      This works in conjunction with an instance created by <Code>createAlertsManager</Code> function below. You need to
-      export <Code>alertsManager</Code> instance and then you can import it in child components in order to trigger
-      alert:
-    </Text>
-
-    <CodeSnippet>
-      {`
-        import { alertsManager } from '../App';
-
-        // ...
-
-        const alert = {...}; // alert props
-        alertsManager.add(alert);
-      `}
-    </CodeSnippet>
-
-    <H2>createAlertsManager</H2>
-
-    <Text>
-      The <Code>createAlertsManager</Code> function returns an instance for managing which alert to display.
-    </Text>
-
-    <CodeSnippet showControls={false}>
-      {/* jsx-to-string:start */}
-      const alertsManager = createAlertsManager();
-      {/* jsx-to-string:end */}
-    </CodeSnippet>
-
-    <AlertsManagerAddMethodList />
-
-    <AlertsManagerRemoveMethodList />
-
-    <AlertsManagerClearMethodList />
-
-    <AlertsManagerSubscribeMethodList />
-  </>
-);
+const AlertPage = () => {
+  const items = [
+    {
+      id: 'examples',
+      title: 'Examples',
+      render: () => (
+        <>
+          <Panel>
+            <Text>
+              Alerts are non-disruptive messages that appear at the top right of the window and provide quick, at a
+              glance feedback on the outcome of an action.
+            </Text>
+            <CodePreview scope={{ alertsManager }}>
+              {/* jsx-to-string:start */}
+              {function Example() {
+                const alert = {
+                  header: 'Optional Headline',
+                  messages: [
+                    {
+                      text: 'Required description copy.',
+                      link: {
+                        text: 'Optional Link',
+                        href: '#',
+                      },
+                    },
+                  ],
+                  type: 'success',
+                  onClose: () => null,
+                } as AlertProps;
+
+                return <Button onClick={() => alertsManager.add(alert)}>Trigger Alert</Button>;
+              }}
+              {/* jsx-to-string:end */}
+            </CodePreview>
+          </Panel>
+        </>
+      ),
+    },
+    {
+      id: 'code',
+      title: 'Code',
```

Comment:
We should change all of these to `Props`:
```suggestion
      id: 'props',
      title: 'Props',
```